api/diagnosis/diagnosis_routes.py

```
@@ -0,0 +1,88 @@
+from api.frame.TimedQueue import TimedQueue
+from api.frame.frame_routes import uid_queues, get_or_create_queue
+
+import cv2
+import numpy as np
+from datetime import datetime
+
+router = APIRouter()
+logger = get_logger(__name__)
+
+@router.get("/api/diagnosis/drowiness")
+async def get_diagnosis_result(request: Request, device_uid: str = Query(..., alias="deviceUid")):
```

Ruff (0.11.9)

Missing return type annotation for public function `get_diagnosis_result`

(ANN201)